examples/model_selection/grid_search_digits.py

```diff
@@ -60,9 +60,11 @@
     print()
     print("Grid scores on development set:")
     print()
-    for params, mean_score, scores in clf.grid_scores_:
+    means = clf.results_['test_%s_weighted_mean' % score]
```

Comment:
I'm not sure but I feel you could argue the same for `test mean_precision weighted`